docs/reference/generated/kubelet.md

```diff
@@ -36,7 +36,7 @@ kubelet
 ### Options
 
 ```
-      --address ip                                               The IP address for the Kubelet to serve on (set to 0.0.0.0 for all interfaces) (default 0.0.0.0)
+      --address ip                                               The IP address for the Kubelet to serve on (set to 0.0.0.0 for all IPv4 interfaces and :: for all IPv6 interfaces) (default 0.0.0.0)
```

Comment:
should be fixed upstream here: https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/options/options.go#L385